lib/portage/dep/soname/multilib_category.py

```diff
@@ -77,6 +78,7 @@
     EM_68K: "m68k",
     EM_AARCH64: "arm",
     EM_ALPHA: "alpha",
+    EM_AMDGPU: "AMDGPU",
```

Comment:
Other values in this dictionary are lower-case, and this one probably also should be:
`EM_AMDGPU: "amdgpu",`

Reply:
Fixed